account-kit/react/src/components/dialog/dialog.tsx

```diff
@@ -40,27 +51,31 @@ export const Dialog = ({ isOpen, onClose, children }: DialogProps) => {
     setScrollLocked(getComputedStyle(document.body).overflow !== "hidden");
   }, []);
 
-  return isOpen
-    ? createPortal(
-        <RemoveScroll enabled={isScrollLocked}>
-          {/* Overlay */}
+  return createPortal(
+    <RemoveScroll enabled={isScrollLocked}>
+      {/* Overlay */}
+      <div
+        aria-modal
+        role="dialog"
+        className={`fixed inset-0 bg-black/80 flex items-end md:items-center justify-center z-[999999] ${
+          !showBackdrop
+            ? "animate-fade-out invisible"
+            : "animate-fade-in visible"
```

Comment:
I actually am not seeing this animation run at all

Reply:
Yeah! I'd fix this! Seen what the issue is